test/ampycloud/test_core.py

```diff
@@ -95,24 +95,24 @@ def test_run_single_point():
     """ Test the code when a single data point is fed to it. """
 
     # Set no lower limits to compute oktas
-    dynamic.OKTA_LIM0 = 0
+    dynamic.MAX_HITS_OKTA0 = 0
 
     # Let's create some data with a single valid point
     data = pd.DataFrame([['1', -100, 2000, 1], ['1', -99, np.nan, 0]],
                         columns=['ceilo', 'dt', 'alt', 'type'])
     # Set the proper column types
     for (col, tpe) in hardcoded.REQ_DATA_COLS.items():
-        data.loc[:, col] = data.loc[:, col].astype(tpe)
+        data[col] = data.loc[:, col].astype(tpe)
 
     # Run the code
     out = run(data)
-    assert out.metar_msg() == 'SCT020'
+    assert out.metar_msg() == 'NCD'
```

Comment:
What would be a minimal test case with an output which is not NCD but actually some cloud layers identified?